src/notification.js

```diff
@@ -144,7 +144,7 @@ export class NotificationElement extends LitElement {
             </a>
           </div>
           <div class="content">
-            Some features may not be yet available in the publised stable
+            Some features may not be yet available in the published stable
```

Comment:
Noticed this on the other PR, word order seems off:

```suggestion
            Some features may not yet be available in the published stable
```